docs/molgenis/README.md

```diff
@@ -1,20 +1,23 @@
 
-MOLGENIS is the world's most customisable platform for (scientific) data and FAIR principles (findability, accessbility,
-interoperability and reusability). Use MOLGENIS to precisely model your data as a schema of tables, columns and
-relationships. Then [MOLGENIS](http://molgenis.org)
-auto generates a complete database application with advanced data entry forms, powerful data up/download options and
-flexible query tools to help you collect, analyse and share your data. If you want to program, you can use PostgreSQL,
-GraphQL api, batch web services or linked data RDF/TTL interface to query/update the data, and use VueJS to create your
-own 'apps'.
+For developers, you can use PostgreSQL, GraphQL api, batch web services or linked data RDF/TTL interface to query/update the data, and use VueJS to create your own 'apps'.
+
+To get started with MOLGENIS, we have created several guides.
+
+| Guide | Description |
+|:------|:-------------|
+| [User guide](./use) | reference for how to create and interact with MOLGENIS databases |
+| [Installation Guide](./run) | step-by-step guide for installing and configuring the MOLGENIS software |
+| [Developers Guide](./dev) | learn about the architecture of MOLGENIS, how to contribute to the code base, how to build custom vue applications, and more |
 
 ## Differences with previous version of MOLGENIS
 
-Core differences with molgenis/molgenis include that it is simpler to setup and operate, it allows multiple data
-schemas, it has more sophisticated data modelling features, and for developers it presents data using a self-documenting
-graphql api. Obviously it doesn't have all features of molgenis/molgenis yet, so that is being maintained as well until
-adequate migration paths exist.
+[molgenis-emx2](http://github.com/molgenis/molgenis-emx2) (or EMX2) is the newest iteration of the MOLGENIS platform. It builds on the previous version ([molgenis/molgenis](https://github.com/molgenis/molgenis), or EMX1) by:
+
+- **Simpler setup and operation**: We simplified the core structure and EMX2 is built using PostgreSQL, Java, GraphQL, and Vue.
+- **Enhanced data modelling features**: we enabled the option for multiple data schemas, schema templates, and FAIR data improvements (e.g., the `ONTOLOGY` data type). For developers, the data API specification is now auto-generated. Any changes to the schema will be automatically documented in the GraphQL API
+- **Improved data APIs**: In EMX2, we've incorporated [GraphQL](https://graphql.org). This allows users to write more advanced queries
 
-Use the menu on the left to navigate through this document.
+We are still in the process of adding EMX1 features into EMX2. For now, EMX1 will be maintained but not actively developed.
```

Comment:
Maybe it is good to also add that we plan to deprecate it at some point in the future?

Reply:
This is good point. I added more information on this